icefall/rnn_lm/train.py

```diff
@@ -589,10 +663,20 @@ def run(rank, world_size, args):
         lr=params.lr,
         weight_decay=params.weight_decay,
     )
+    scheduler = NewBobScheduler(optimizer)
```

Comment:
Could you make it possible for users to pass some arguments to the scheduler?

Reply:
Sure.